src/test/java/IMDbTop250Scraper.java

```diff
@@ -0,0 +1,48 @@
+import com.codeborne.selenide.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static com.codeborne.selenide.Selenide.*;
+
+public class IMDbTop250Scraper {
+
+    public static void main(String[] args) {
```

Comment:
As I mentioned a bit earlier this is hard to read, if a month later I would ask you to explain what is happening here, I think instead of 10 seconds it will take you minutes..

I suggest using small methods, each method does something that it should, just like you did in previous PR.

`
openImdbPage()
selectOptionFromMenu()
doThis
doThat....
`

Reply:
Let's clear code so it will be readable and maintainable easily, then let's see how it works, does it suit you? :)